rks-openapi.yaml

```
@@ -220,6 +220,28 @@ paths:
           description: "Forbidden. Invalid X-Vault-Token"
         '404':
           description: Group not found
+  '/rks/v1/group/{groupname}/nodes/{nodeId}':
+    delete:
+      summary: Revoke a Node by revoking its token
+      description: revoke token for node of **groupname** with nodeId like **X-LCDN-nodeId**,  
+        **adminToken** is used as X-Vault-Token for authentication. Note that this call will
+       only revoke Node Token, that's to say, if you want to disallow this node to register again,
+       there is two ways. 
+       Makes callback url answer "40X" for this nodeId
+       Renew grouptoken and spread it to all nodes, execept this one 
```

reformulation suggestion
```suggestion
      description: |
        revoke the **nodeToken** for a node part of group **groupname** with nodeId **X-LCDN-nodeId**, 
        **adminToken** is used as X-Vault-Token for authentication.
        Note that this call will revoke the **nodeToken** but won't forbid the node to register again. 
        To forbid node registration there are two ways:
        Make group manager callback url answer "40X" for this nodeId
        Renew the group grouptoken and spread it to all nodes, except this one 
```